server/repositories/CaptureRepository.js

```diff
@@ -48,6 +48,23 @@ class CaptureRepository extends BaseRepository {
       );
       delete filterObject.captured_at_end_date;
     }
+    if (filterObject.matchting_tree_distance || filterObject.matchting_tree_time_range) {
+      const knex = this._session.getDB();
+      result.where(`id`, 'in', knex.raw(`
+        select 
+          distinct(tc.id)
+        from capture tc 
+        JOIN tree tt ON
+          ST_DWithin(
+            tc.estimated_geographic_location,
+            tt.estimated_geographic_location,
+          ${filterObject.matchting_tree_distance})
```

Comment:
if matching_tree_distance is null or undefined, would that still work? Seems like you are not adding that null check